_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
```

Comment:
things like this might need `{{ tag | url_encode }}` in certain scenarios, just FYI